packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -1,9 +1,10 @@
+import 'dart:developer';
```

Comment:
wow what class requires this? never seem it before

Reply:
This is for logging the stack-trace in line 337. It won't be needed after we upgrade to Dart 2.16 (which is in Flutter 2.10.0, released on 2022-02-03).